ethwrite/src/config.ts

```diff
@@ -0,0 +1,19 @@
+import { util } from '@chainlink/ea-bootstrap'
+
+export type Config = {
+  rpcUrl: string
+  network?: string
+  privateKey: string
+  api: any
```

Comment:
I think you can remove `api` right?

Reply:
If I want to use `ExecuteWithConfig<C extends Config>`, I can't, since it is required in main Config type. And I don't feel like changing the `adapter.ts` boilerplate, since IMHO should be removed at some point.

I was thinking of changing this type to ExecuteWithConfig<C> but I am considering of checking this again separately, maybe as part of https://github.com/smartcontractkit/external-adapters-js/pull/293.

I can also just use `execute = async (input: AdapterRequest, config: Config): Promise<AdapterResponse>` for the particular adapter as in `synth-index` though. Thoughts?

Reply:
You're right, the tweak is made is some places is defining the makeExecute as `const makeExecute = (config?: Config): Execute => {...`

Reply:
Let's check changing it to either ExecuteWithConfig<C> or removing the requirement of `api` param, in another PR, what do you think?

Reply:
Let's leave it as it is. There's an issue open on `Config` inflexibility, we can leave it for another PR

Reply:
Tagging #217